### ISSUE TITLE
[ARM] support

### ISSUE THREAD

**Issue by dudo** [pixie-io/pixie#1752]
The [website](https://docs.px.dev/installing-pixie/requirements/#cpu) shows that ARM is not supported, but this [issue](https://github.com/pixie-io/pixie/issues/147#issuecomment-1451010612) (and linked issues) all seem to be resolved.

Is is safe to say that ARM is supported now?

**Comment by ddelnano** [comment]
Hi @dudo, yes ARM is supported today and we have a PR open to update our docs (https://github.com/pixie-io/docs.px.dev/pull/259). We should hopefully be merged that soon.

**Comment by rmadams2** [comment]
@ddelnano is ARM64 supported yet?

**Comment by ddelnano** [comment]
@rmadams2 yes, it is supported as indicated by the documentation fix I mentioned above. If you are aware of any issues, please file a GitHub issue for them so we can prioritize any fixes.

**Comment by rmadams2** [comment]
@ddelnano Apologies, and pardon me if I'm misunderstanding, but aren't ARM and ARM64 architectures different? I didn't know if when you're saying 'ARM' it includes both 32 and 64 bit architectures.

**Comment by ddelnano** [comment]
You are correct that the unsuffixed name usually refers to the 32 bit architecture. In the context of our GitHub issues, we haven't been precise in stating ARM64. All of our testing has been done on ARM64 and I don't believe we've ever tested 32 bit arm.
